x11-themes/papirus-icon-theme/papirus-icon-theme-20211001-r1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg
+
+DESCRIPTION="Free and open source SVG icon theme"
+HOMEPAGE="https://github.com/PapirusDevelopmentTeam/papirus-icon-theme"
+SRC_URI="https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="pantheon"
+
+src_compile() { :; }
+
+src_install() {
+	insinto /usr/share/icons
+	doins -r Papirus{,-Dark,-Light}
+	use pantheon && doins -r ePapirus{,-Dark}
```

Comment:
In general we don't add USE flags to control installing few kb worth of stuff, 
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301

but even without that, why would you not want to install these variants unconditionally? :P

Reply:
I'm fine to install those unconditionally. I simply was always confused what exactly ePapirus theme is (it's not very obvious from its name), and according to README that flavour is sensible for the pantheon desktop environment only.

@juippis do you think we should remove use flag?

Reply:
I've opened a new PR with new version of the package and unconditional theme installation (but added a comment in the ebuild what those themes are) :D

Reply:
But, I think we need the use flag :P